src/api/lib/activexml/node.rb

```diff
@@ -447,7 +447,7 @@ def value( symbol )
         return @value_cache[symbols] = elem.first.inner_text
       end
 
-      return @value_cache[symbols] = nil
+      @value_cache[symbols] = nil
```

Comment:
I would prefer to have two seperate statements, one for assignment and one for returning
```
@value_cache[symbols] = nil
@value_cache
```